### PR TITLE
Python 3: Update commands module

### PR DESCRIPTION
**Pull request by hsmj1412**
Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

**Comment by hsmj1412** [comment]
Depend on https://github.com/avocado-framework/avocado/pull/2510

File: virttest/nfs.py [hunk 1]
```diff
@@ -285,8 +285,9 @@ def is_mounted(self):
                                                                 self.mount_dir)
         cmd = self.ssh_cmd + "'%s'" % find_mountpoint_cmd
         logging.debug("The command: %s", cmd)
-        if process.system(cmd, verbose=True, ignore_status=True, shell=True):
-            return False
+        status, output = process.getstatusoutput(cmd)
+        if status:
+            logging.debug("The command result: <%s:%s>", status, output)
```
Comment (kylazhang) [inline virttest/nfs.py]: Why change the logical here? No need to return False?

File: scripts/regression.py [hunk 2]
```diff
@@ -19,9 +18,16 @@
...
+def getstatusoutput(cmd):
+    """Return (status, output) of executing cmd in a shell."""
+    pipe = os.popen('{ ' + cmd + '; } 2>&1', 'r')
+    text = pipe.read()
+    sts = pipe.close()
+    if sts is None:
+        sts = 0
+    if text[-1:] == '\n':
+        text = text[:-1]
+    return sts, text
```
Comment (kylazhang) [inline scripts/regression.py]: If no use for the status, just define with getoutput and only return output is ok.

**Comment by hsmj1412** [comment]
1. commands module redirect stderr to stdout, but subprocess could not do it
2. the output of subprocess is bytes in Python3

**Comment by kylazhang** [comment]
And it is better to add the explanation in commit msg

File: docs/source/conf.py [hunk 3]
```diff
@@ -16,7 +16,18 @@
 
 import os
 import sys
-import subprocess
+
+
+def getstatusoutput(cmd):
```
Comment (chunfuwen) [inline docs/source/conf.py]: I don't think we need create new method, moreover, no need redirect stderror to stdoutput

File: virttest/nfs.py (hunk 1 above)
Comment (hsmj1412) [reply virttest/nfs.py]: Fixed

File: scripts/regression.py (hunk 2 above)
Comment (hsmj1412) [reply scripts/regression.py]: Updated

File: docs/source/conf.py (hunk 3 above)
Comment (hsmj1412) [reply docs/source/conf.py]: I can not confirm if redirection is necessary or not.

**Pull request merged**